koku/api/report/all/openshift/serializers.py

```diff
@@ -0,0 +1,100 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""OCP-on-AWS Report Serializers."""
+
+import api.report.aws.serializers as awsser
+import api.report.ocp.serializers as ocpser
+from api.report.serializers import validate_field
+
+
+class OCPAllGroupBySerializer(awsser.GroupBySerializer,
+                              ocpser.GroupBySerializer):
```

Comment:
So for clarity did we normalize the Azure data into the AWS format for the ALL set and that is why we only need these two serializers?

Reply:
Nevermind. This answers the above question:
https://github.com/project-koku/koku/pull/1558/files#diff-d69a5b7ac851065a12b0330f6328d887R174